docs/contributing/mkdwn-styles.md

```diff
@@ -89,7 +89,7 @@ Reference-style: ![alt text][logo]
 
 Images from any folder can be used by providing path to file. Path should be relative to markdown file.
 
-![img](../../../static/img/logo.svg)
+![img](../../static/img/logo.svg)
```

Comment:
Oh, I think I planned to just delete this file since I moved most of the contents over to `style-guide.md` page, but this info definitely needs to be added to that.

Reply:
ok, deleted the file. Everything in there is available in a link in the Markdown section.